src/main/java/Duke.java

```diff
@@ -18,16 +17,10 @@ public static void main(String[] args) {
                 break;
             }
             else if (userInput.equals("list")){
-                System.out.println("Here are your tasks:");
-                for (int j = 0; j<count; j++) {
-                    System.out.println(j+1 + ". " + tasks[j]);
-                }
+                listTasks(tasks, count);
             }
             else if (userInput.startsWith("done")){
-                int itemNumber = Integer.parseInt((userInput.substring(5)));
-                tasks[itemNumber-1].markAsDone();
-                System.out.println("Nice! I've marked this task as done:");
-                System.out.println("  " + tasks[itemNumber-1]);
+                markTaskAsDone(tasks, userInput);
```

Comment:
It would be better if you can consider to wrap these "println"s into markTaskAsDone; similar suggestion for other command case

Reply:
Thank you for your suggestion 👍